firefox-ios/Client/Frontend/Browser/Tabs/Views/TabDisplayView.swift

```diff
@@ -278,6 +278,7 @@ class TabDisplayView: UIView,
     func applyTheme(theme: Theme) {
         self.theme = theme
         collectionView.backgroundColor = theme.colors.layer3
+        collectionView.visibleCells.forEach { ($0 as? ThemeApplicable)?.applyTheme(theme: theme) }
```

Comment:
The cells didn't update on their own otherwise? **In theory** they should 🤔

Reply:
I agree but testing without this line causes theming issues + it happens intermittently. You can see the issue more commonly performing these steps.

1. Navigate to private panel
2. Add a private tab and then close the private tab
3. Navigate to normal tab panel
4. See collection view 

Seems like a case where `applyTheme` is called directly and not using Themable notifications, so we need to apply theme for subviews here.